test/unit-tests/vast_test.js

```diff
@@ -1721,4 +1761,281 @@ describe('ad_manager_vast', function() {
     vastAdManager.loadMetadata({"tagUrl": "http://blahblah"}, {}, content);
     expect(vastAdManager.vastUrl).to.be("http://blahblah");
   });
+
+  it('VPAID 2.0: Should parse VPAID linear creative', function() {
+    vpaidInitialize();
+    var ad = amc.timeline[0];
+    expect(ad).to.be.an('object');
+    expect(ad.duration).to.eql(16);
```

Comment:
Not really sure we should check for strings, the duration tag is like this: `<Duration>00:00:16</Duration>` 

Added a validation for the case: `<Duration>16</Duration>`